tests/generate_code.bat

```diff
@@ -55,7 +55,7 @@ set TEST_NOINCL_FLAGS=%TEST_BASE_FLAGS% --no-includes --no-fb-import
 ..\%buildtype%\flatc.exe --cpp %TEST_BASE_FLAGS% --cpp-ptr-type flatbuffers::unique_ptr native_type_test.fbs || goto FAIL
 
 @rem Generate the optional scalar code for tests.
-..\%buildtype%\flatc.exe --kotlin --rust --lobster optional_scalars.fbs || goto FAIL
+..\%buildtype%\flatc.exe --csharp --java --kotlin --rust --lobster optional_scalars.fbs || goto FAIL
```

Comment:
@paulovap  I added --java to the .bat file of the generation script. Just wanted to give you a heads up.

Reply:
Thanks! I guess I missed that one.